.swift-version

```diff
@@ -0,0 +1 @@
+DEVELOPMENT-SNAPSHOT-2023-11-20-a
```

Comment:
Is this for the benefit of `swiftenv`, or does any of our CI tooling need this?

Reply:
Ah yeah but it's my local one I'll add it to the .gitignore if testing shows we don't need it